### PR TITLE
Ra/loc/split versions

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@hishamco, I've moved #18556 here so you'll get credit for your commit. I'll ping you if I have any questions.

### CODE REVIEW

**Comment by hishamco** [comment]
No problem Rick, I will review the PR ASAP

**Comment by Rick-Anderson** [comment]
@hishamco  NO, not ready for review. I'm just fixing some snippet path errors.

File: aspnetcore/fundamentals/localization.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: Globalization and localization in ASP.NET Core
 author: rick-anderson
 description: Learn how ASP.NET Core provides services and middleware for localizing content into different languages and cultures.
 ms.author: riande
+ms.date: S
```
Comment (hishamco) [inline aspnetcore/fundamentals/localization.md:6]: Why we need this, while we already have `ms.date`

**Comment by hishamco** [comment]
Shall I start on POLocalization?

**Comment by Rick-Anderson** [comment]
Please do. These changes will be live for you to review in 10 minutes.

Thanks for your great work!

**Comment by linksnail** [comment]
Localization dosen't work well untill move SharedResource.cs out of Resources

**Comment by hishamco** [comment]
Let me double check, because last time I make sure that the localization works fine

Thanks for reporting this

**Comment by hishamco** [comment]
You are right @linksnail, it should be at the root so it can be discoverable

I create a PR #19905 to fix it

Thanks

**Pull request merged**